src/libasr/codegen/x86_assembler.h

```diff
@@ -345,6 +345,10 @@ class X86Assembler {
         return m_asm_code;
     }
 
+    void append_asm(std::string asm_code) {
+        m_asm_code += asm_code;
+    }
+
```

Comment:
The `X86Assembler` already has an `emit()` function which adds text to `m_asm_code`, but it is defined as private so we are unable to call it using the assembler object `m_a`. I am unsure if we should make it public. I currently defined this `append_asm()` to allow adding text to `m_asm_code`.

Reply:
I would remove append_asm. I would put all the header right underneath of:
```
        m_asm_code = "BITS 32\n\n";
```

Reply:
Yes, got it. I removed the `append_asm()` function.